src/linker/Linker/LoggingReflectionPatternRecorder.cs

```diff
@@ -44,7 +44,8 @@ public void RecognizedReflectionAccessPattern (MethodDefinition sourceMethod, In
 
 		public void UnrecognizedReflectionAccessPattern (MethodDefinition sourceMethod, Instruction reflectionMethodCall, IMetadataTokenProvider accessedItem, string message)
 		{
-			_context.LogMessage (MessageImportance.Low, message);
+			_context.LogMessage (MessageContainer.CreateWarningMessage (message, 2006, "Unrecognized reflection pattern",
+				MessageOrigin.TryGetOrigin (sourceMethod, reflectionMethodCall != null ? reflectionMethodCall.Offset : 0)));
```

Comment:
What are the cases where the reflectionMethodCall is null? Similar to above, I'd avoid reporting offset 0 if we don't have the actual origin.